src/platforms/dialogflow/facebook/FacebookReply.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2019 Rain Agency <contact@rain.agency>
+ * Author: Rain Agency <contact@rain.agency>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import * as _ from "lodash";
+import { IBag, IVoxaEvent } from "../../../VoxaEvent";
+import { addToSSML, addToText, IVoxaReply } from "../../../VoxaReply";
+import { DialogflowReply } from "../DialogflowReply";
+import { FacebookEvent } from "./FacebookEvent";
+
+export interface IFacebookPayload {
+  facebook: {
+    attachment?: {
+      payload: any;
+      type: string;
+    };
+    quick_replies?: any[];
+    text?: any;
+  };
+}
+
+export class FacebookReply extends DialogflowReply {
```

Comment:
override the payload type to make it `public payload: IFacebookPayload`

Reply:
It is public and it's already there:
https://github.com/VoxaAI/voxa/pull/200/files/bd0a4389c665f3e31c62f9de1d97ff4cf8d6d40a#diff-60f2950f06e7c6e0a52248fea00a9287R62